index.src.html

```diff
@@ -1442,15 +1452,9 @@
     `false`.
 2. If {{VideoDecoderConfig/codedWidth}} = 0 or
     {{VideoDecoderConfig/codedHeight}} = 0, return `false`.
-3. If {{VideoDecoderConfig/cropWidth}} = 0 or {{VideoDecoderConfig/cropHeight}}
-    = 0, return `false`.
-4. If {{VideoDecoderConfig/cropTop}} + {{VideoDecoderConfig/cropHeight}} >=
-    {{VideoDecoderConfig/codedHeight}}, return `false`.
-5. If {{VideoDecoderConfig/cropLeft}} + {{VideoDecoderConfig/cropWidth}} >=
-    {{VideoDecoderConfig/codedWidth}}, return `false`.
-6. If {{VideoDecoderConfig/displayWidth}} = 0 or
-    {{VideoDecoderConfig/displayHeight}} = 0, return `false`.
-7. Return `true`.
+3. If {{VideoDecoderConfig/displayAspectWidth}} = 0 or
```

Comment:
Can you add a check that if one is provided, the both are

Reply:
I'm not sure I follow. The aspect ratio is not marked as optional, so this already makes sure we have both dAW and dAH in all cases.

Reply:
dictionary members are implicitly optional. you can mark them "required", but we like optional for these.

Reply:
Ah, thank you! Updated.